dev/com.ibm.ws.security.kerberos.java8/src/com/ibm/websphere/security/s4u2proxy/SpnegoHelper.java

```diff
@@ -72,19 +72,18 @@ public class SpnegoHelper {
 
     /**
      * Build a SPNEGO Authorization string using a Kerberos credential from the delegate service that impersonates the user (S4U2self).
-     * The method will use that credential to request a SPNEGO token for a targetServiceSpn (SPN) for the target service system.
+     * The method will use that credential to request a SPNEGO token for a target service principal name  for the target service system.
      * If the system property java.security.krb5.conf has not been set by the run time, you need to set the java.security.krb5.conf that point to your Kerberos configuration file.
      *
-     * @param upn - UserPrincipalName of the user for which the SPNEGO token will be generated.
-     * @param targetServiceSpn - ServicePrincipalName of system for which SPNEGO token will be targeted.
-     * @param lifetime - Lifetime for the context, for example GSSCredential.INDEFINITE_LIFETIME
+     * @param upn – User principal name of the user for which the SPNEGO token will be generated.
+     * @param targetServiceSpn – Service principal name of a system for which SPNEGO token will be targeted.
+     * @param lifetime – Life time for the context, for example GSSCredential.INDEFINITE_LIFETIME.
```

Comment:
lifetime is actually one word, so we should keep it how ti was before.
```suggestion
     * @param lifetime – Lifetime for the context, for example GSSCredential.INDEFINITE_LIFETIME.
```